app/models/project.rb

```diff
@@ -0,0 +1,6 @@
+class Project < ApplicationRecord
+  has_many :project_teams
+  has_many :teams, through: :project_teams
+
+  has_many :project_users
```

Comment:
Do we not need association to users here?

Reply:
user model is not created yet... so thats why i did not add it here